components/learn/modules/ROOT/pages/deploy-and-interact.adoc

```diff
@@ -186,17 +186,147 @@ TIP: To learn more about `send-tx` and `call`, check out the https://docs.openze
 [[interacting-programatically]]
 == Interacting Programatically
 
-from sdk interacting guide (https://docs.openzeppelin.com/sdk/2.6/interacting#setup)
+While interacting via the command line is useful for running quick tests, in the end you will want to interact with your contracts from your own code. Here, we will see how to use https://web3js.readthedocs.io/en/1.2.4/[web3.js] to do so, so let's install it.
 
-replace manual setup with contract loader?
+[source,console]
```

Comment:
I've been using triple backticks, but wouldn't mind switching to this style, as long as all guides are consistent.

Reply:
Does it really matter?